src/detectors/ZDC/ZDC.cc

```diff
@@ -195,6 +195,13 @@ extern "C" {
           app   // TODO: Remove me once fixed
         ));
 
+        app->Add(new JOmniFactoryGeneratorT<FarForwardNeutronReconstruction_factory>(
+           "HcalFarForwardZDCNeutronCandidates",
+          {"HcalFarForwardZDCClusters"},  // edm4eic::ClusterCollection
+          {"HcalFarForwardZDCNeutronCandidates"}, // edm4eic::ReconstrutedParticleCollection,
+          app   // TODO: Remove me once fixed
+        ));
+
```

Comment:
Whoops! Should've commented on this in my last review: alongside moving the algo/factory to reco, I would suggest moving this to the reco plugin (`global/reco/reco.cc`) and renaming the output to `ReconstructedFarForwardNeutrons` (or similar).

Reply:
Done, thanks!

Reply:
Have you pushed the changes? I'm still seeing the factory declaration in `zdc.cc`...

Reply:
oops; made the change locally but forgot to git add; git commit ..., before doing git pull; git push.   Pushed to the github now.

Reply:
Thanks! One final suggestion and then I'm happy to merge! I'd suggest renaming the output to `"ReconstructedFarForwardZDCNeutrons"` just to make it consistent with most of the other reconstructed particle collections

Reply:
done